core/src/main/java/org/elasticsearch/action/ingest/IngestActionFilter.java

```diff
@@ -101,6 +101,7 @@ void processIndexRequest(Task task, String action, ActionListener listener, Acti
     }
 
     void processBulkIndexRequest(Task task, BulkRequest original, String action, ActionFilterChain chain, ActionListener<BulkResponse> listener) {
+        long ingestStartTime = System.currentTimeMillis();
```

Comment:
any time we measure deltas (how long something took) we should use nanoTime. currentTimeMillis can be reset by ntp ... if the bulk api uses it for this purpose, it should be fixed...